exampleCourse/questions/demo/overlayDropdown/question.html

```diff
@@ -10,7 +10,7 @@
 
 <pl-overlay width="700" height="700" clip="false">
     <pl-background>
-        <img src="{{options.client_files_question_url}}/europe_annotated.png" style="max-width: 100%; max-height: 100%">
+        <img alt="annotated map of Europe" src="{{options.client_files_question_url}}/europe_annotated.png" style="max-width: 100%; max-height: 100%">
```

Comment:
I hit https://github.com/PrairieLearn/PrairieLearn/issues/11841 before I was able to resolve all of these.